pom.xml

```diff
@@ -647,7 +647,7 @@
             <dependency>
                 <groupId>io.netty</groupId>
                 <artifactId>netty-handler</artifactId>
-                <version>${netty.version}</version>
+                <version>4.1.94.Final</version>
```

Comment:
Execuse me, byt why not change the `${netty.version}` to `4.1.94.Final` directly?

Reply:
Maybe you can add a new variable like `${netty.handler.version}` instead of directly using it in the dependency version.

Reply:
Got it, I'll go modify it.

Reply:
it's better to update `netty.version` to `4.1.94.Final ` other than only `handler`.